.gitignore

```diff
@@ -1,8 +1,38 @@
 /nbproject/private/
 /build/
 /dist/
+/bin/
 *.class
+/.idea
+.classpath
+.project
+manifest.mf
+
+notepad.iml
 
 # Mac OS System
 .DS_Store*
 ._*
+
+# Compiled class file
+*.class
```

Comment:
`*.class` is already present [on line 5](https://github.com/pH-7/Simple-Java-Text-Editor/blob/4e6608ea92095fb9be467f23a57ce177d88c34a8/.gitignore#L5).